content/english/episodes/ep-8-trailer.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Trailer: Episode 8 with Mojdeh Rastgoo"
+meta_title: "Trailer: Episode 8 with Mojdeh Rastgoo"
+description: "A preview of our chat with Mojdeh Rastgoo. Watch the full episode on April 15, 2025"
+date: 2025-04-10T00:00:00-08:00
+host: ["Tereza", "Georgi"]
+draft: false
+tags: ["python", "iran", "france", "pyladies", "psf-coc-wg", "pypodcats", "community", "project manager"]
+publish_date: 2025-04-10T00:00:00-08:00
+podcast_file: "/audio/hidden-figures-ep8-trailer.mp3"
+podcast_duration: 98
+episode_image: "images/ep8-mojdeh.png"
+podcast_bytes: "3889257" # the length of the episode in bytes
+images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
+explicit: false 
+type: 'episode'
+episode: '8'
+season: 1
+transcript: '/transcript/hidden-figures-of-python-ep8-trailer.srt'
+episode_type: trailer
+layout: episode
+youtube_id: "qetbNfnCkJM"
+  
+speaker:
+  url: "/speakers/mojdeh-rastgoo/"
+  title: "Mojdeh Rastgoo"
+  image: "/images/mojdeh_rastgoo.jpeg"
+  description: "
+Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Pris.
+She has a PhD in computer sicence and has been an active memnber of the community since 2018. 
+She is a co organizer of PyLadies Paris and member of PSF Code of Conduct Working-Group since mid 2023.
+And recently she has joined the PyPodcats team as the fifth organizer.  
+
+She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon.DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 
```

Comment:
```suggestion
She participated in organization of EuroSciPy 2023/2024 conference, JupyterCon 2023, PyCon DE & PyData Berlin 2024, and PyLadies Paris meetups since 2022. 
```

Reply:
Should be PyCon DE instead of PyCon.DE?